src/app/components/tree-view/index.html

```diff
@@ -49,23 +53,23 @@ <h3 class="sky-text-success">Use when</h3>
       <h3>Selection</h3>
         <sky-docs-visual-example>
           <sky-docs-visual-example-text>
-            <p>You can use single-select or multiselect mode when you use a tree view in a selection context.</p>
+            <p>You can use the single-select and multiselect modes to allow users to select list items.</p>
           </sky-docs-visual-example-text>
           <sky-docs-visual-example-image>
             <img src="~/assets/img/guidelines/angular-tree-view/option-select.png" />
           </sky-docs-visual-example-image>
         </sky-docs-visual-example>
         <sky-docs-visual-example>
           <sky-docs-visual-example-text>
-            <p>With cascading selection, when users select a node the descendant nodes are also selected. Use this mode if it's likely that users will want to select a node and all its descendants.</p>
+            <p>You can use the cascading selection mode to select descendant nodes when users select items. Use this mode when users are likely to select all descendants within nodes.</p>
```

Comment:
Should cascading selections really be listed as an option in our design docs? As far as I can tell, it’s not an option that is set through our components, so should we really call it out as an option when we don’t call out the other options for the components that we are building on top of? Same question for the way we highlight it in the demo.

Reply:
You are correct - its not something we had to build, as it was already supported by the 3rd-party library. With that said, I think its important to still talk about best practices in design, even if its not a particular function of our component. These are scenarios that our consumers will come across and have question about. That's just my 2 cents.

Reply:
After @Blackbaud-ToddRoberts 's comment above how this page is more about the "Tree view" pattern in general, and NOT the specific implementation, I think it further supports keeping UX guidelines like this in.

Reply:
Works for me. Just wanted to make sure. I'll resolve this conversation.